lms/djangoapps/grades/admin.py

```diff
@@ -26,6 +27,49 @@ class CoursePersistentGradesAdmin(KeyedConfigurationModelAdmin):
         }),
     )
 
+
+class PersistentSubsectionGradeOverrideAdmin(admin.ModelAdmin):
```

Comment:
Question: Will this allow you to add additional entries, or just read existing ones?

Reply:
<img width="1220" alt="screen shot 2017-11-23 at 12 02 32 am" src="https://user-images.githubusercontent.com/4252738/33145285-a67b2438-cfe1-11e7-8d01-8835c0e9ad66.png">
Yes, we can create new entries as well using this.